clarity_ext/dilution.py

```diff
@@ -72,7 +82,7 @@ def _build_plate_sorting_map(containers):
         # Fetch an unique list of container names from input
         # Make a dictionary with container names and plate position sort numbers
         unique_containers = sorted(list(
-            {container.id for container in containers}))
```

Comment:
I don't understand this change. container.id is defined within the container class as container.resource.id